content/200-concepts/100-components/02-prisma-client/000-working-with-prismaclient/010-generating-prisma-client.mdx

```diff
@@ -123,7 +123,7 @@ import { PrismaClient } from '../prisma/src/generated/client'
 
 #### Importing Prisma Client
 
-By generating Prisma Client into `node_modules/.prisma/client` and exporting it from `node_modules/@prisma/client`, you can import it and instantiate the client in your code as follows:
+By generating Prisma Client into `node_modules/.prisma/client` and exporting it from `node_modules/.prisma/client`, you can import it and instantiate the client in your code as follows:
```

Comment:
```suggestion
By generating Prisma Client into `node_modules/.prisma/client` and exporting it from `@prisma/client`, you can import it and instantiate the client in your code as follows:
```

I think this better capture what this wants to say: We export into that folder, but use the `@prisma/client` package to import which does the redirect under the hood.